daemon/daemon.go

```diff
@@ -136,6 +140,9 @@ func (d *Daemon) RemoveDriver(language string) error {
 
 func (d *Daemon) DriverPool(ctx context.Context, language string) (*DriverPool, error) {
 	d.mu.RLock()
+	if l, ok := d.aliases[language]; ok {
+		language = l
+	}
```

Comment:
Here and below, as every time we override `language` - we actually use it for routing to an appropriate driver, I think logging it with `DEBUG` log level would also make sense. WDYT?

Reply:
Sounds good, let's do it!

Reply:
Added this to the write path instead (when the driver is loaded).